src/Routing/Route/DashedRoute.php

```diff
@@ -56,11 +56,12 @@ protected function _camelizePlugin($plugin)
      * camelBacked form.
      *
      * @param string $url The URL to parse
+     * @param string $method The HTTP method.
      * @return array|false An array of request parameters, or false on failure.
      */
-    public function parse($url)
+    public function parse($url, $method = '')
```

Comment:
Quoted from the first table in the guidelines:
> Add a default argument to an existing method　Yes 


Which does it mean? Adding a default value to an argument?
```php
public function parse($url = 'http://localhost/')
```

Or adding an argument with default value?
```php
public function parse($url, $foo = 'bar')
```